pymc3/distributions/distribution.py

```diff
@@ -322,6 +343,9 @@ def dist(
             rv_out.update = (rng, new_rng)
             rng.default_update = new_rng
 
+        if initval != UNSET and initval is not None:
+            rv_out.tag.test_value = initval
```

Comment:
If there's a user-specified `initval` there's a good chance that's because of numerical reasons.. The `__new__` implementation sets `test_value` too.